admin-ui/plugins/auth-server/plugin-metadata.js

```diff
@@ -176,6 +180,11 @@ const pluginMetadata = {
       path: PLUGIN_BASE_APTH + '/authn',
       permission: SCOPE_READ,
     },
+    {
+      component: LdapAddPage,
+      path: PLUGIN_BASE_APTH + '/authn/ldap/new',
+      permission: SCOPE_WRITE,
+    },
```

Comment:
Use LDAP_WRITE for LDAP add/edit routes  
Replace `permission: SCOPE_WRITE` with `permission: LDAP_WRITE` in admin-ui/plugins/auth-server/plugin-metadata.js for the LDAP add (`/authn/ldap/new`, lines 183–187) and edit routes (lines 71–72).